tools/cli/go-whisk-cli/commands/api.go

```diff
@@ -445,8 +453,11 @@ var apiListCmd = &cobra.Command{
  * Takes an API object (containing one more more single basepath/relpath/operation triplets)
  * and some filtering configuration.  For each API endpoint matching the filtering criteria, display
  * each endpoint's configuration - one line per configuration property (action name, verb, api name, api gw url)
+ * Parses and initializes values for ApiFilteredRow struct
  */
-func printFilteredListApi(resultApi *whisk.RetApi, api *whisk.ApiOptions) {
+func parseFilteredList(resultApi *whisk.RetApi, api *whisk.ApiOptions,flagType string) []whisk.ApiFilteredList {
```

Comment:
The function names is not really representative of the function behavior - which is to generate a filtered list.  There's really not much parsing.